gdal/frmts/raw/byndataset.cpp

```diff
@@ -392,15 +372,13 @@ const char *BYNDataset::GetProjectionRef()
 {
```

Comment:
GetProjectionRef() can be called several times, and if so, currently this will leak pszProjection. So add a preliminary if(pszProjection) return pszProjection;